### PR TITLE
Restructure/rename directories and files

### PR DESCRIPTION
**Pull request by steplica**
* Rename `dto/` package from `Request` to `request`
* Add sub-package to `dto/` called `response`
* Move active site parse logic from `parser` to `parser/activesite`
* Remove `WEB-INF` folder because it's no longer required
* Renamed variables to follow Java conventions
* Cleaned up `pom.xml`
* Optimized import statements
* Correct type `levenstein` to `levenshtein`
* Cleaned up log statements to use Slf4j variable interpolation
* Removed unused PDB/mmCIF protein readers

### CODE REVIEW

File: src/main/java/org/moltimate/moltimatebackend/config/SwaggerConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.time.LocalDate;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfiguration {
+
+    @Bean
+    public Docket moltimateApi() {
```
Comment (blackpan2) [inline src/main/java/org/moltimate/moltimatebackend/config/SwaggerConfiguration.java]: Did we confirm that this still works after moved?

**Comment by blackpan2** [comment]
Fixes #122

File: src/main/java/org/moltimate/moltimatebackend/config/SwaggerConfiguration.java (hunk 1 above)
Comment (steplica) [reply src/main/java/org/moltimate/moltimatebackend/config/SwaggerConfiguration.java]: Yes, it still works

**Comment by steplica** [comment]
@blackpan2  @jmiller656 I updated, merged with dev, and tested. Can y'all check this out to make sure it looks good?

File: src/main/appengine/app.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ runtime: java
 env: flex
 runtime_config:
     jdk: openjdk8
-service: moltimate-backend
+service: default
```
Comment (jmiller656) [inline src/main/appengine/app.yaml]: I think this line can be removed

**Pull request merged**